tests/unit/src/test/scala/tests/markdown/MarkdownFileSuite.scala

```diff
@@ -32,8 +34,21 @@ class MarkdownFileSuite extends FunSuite {
     }
   }
 
-  check(
-    "basic",
+  def checkRenderToString(name: String, original: MarkdownPart, expected: String)(implicit
```

Comment:
Instead of doing a separate check method we can just add an assertion to check if renderToString gives back the original.

Reply:
sure, but .. you know.. Occam's razor
to make tests cleaner i didn't want to mix several aspects together..
sometimes it's cleaner to separate certain aspect from the bigger picture and test it separately

in this case `checkParse` and `checkRenderToString` not only work in opposite directions, but also may require different test data

on top of that.. these are 2 separate functions, right.. let's say tomorrow you would adapt another feature-toggle which would affect only one of them either `parse` or `renderToString`.. 

just.. my five cents, you know

Reply:
please let me know if this is something you absolutely need